R/pkg/R/generics.R

```diff
@@ -63,6 +63,10 @@ setGeneric("countByValue", function(x) { standardGeneric("countByValue") })
 # @export
 setGeneric("crosstab", function(x, col1, col2) { standardGeneric("crosstab") })
 
+# @rdname freqItems
+# @export
+setGeneric("freqItems", function(x, ..., support = 0.01) { standardGeneric("freqItems") })
```

Comment:
I think using vector for cols is more R stylish.
setGeneric("freqItems", function(x, cols, support = 0.01) { standardGeneric("freqItems") })